web-marketplace/src/legacy-pages/Dashboard/Dashboard.tsx

```diff
@@ -468,6 +468,11 @@ export const Dashboard = () => {
                 navigate('/');
                 setMobileMenuOpen(false);
               }}
+              hasOrganization={!!organizationAccount}
+              onCreateOrganization={() => {
+                setMobileMenuOpen(false);
+                navigate('/organizations/create');
+              }}
```

Comment:
I believe we should re-use `createOrganization` from `useOrganizationActions` instead because it handles some edge cases (ie account doesn't have a wallet address associated to it)